doc/pages/command-parsing.asciidoc

```diff
@@ -1,26 +1,24 @@
 = Command Parsing
 
-Kakoune commands, either loaded from a script, or written in the command
```

Comment:
There's a merge commit in your branch - it merges two commits that make identical content changes.
Running `git rebase origin/master` should get rid of the duplicated commit and the merge commit.

Reply:
I did this and pushed the changes. Is it okay now?

Reply:
Nope, you added more redundant commits instead of removing them.
You can see all relevant commits with `git log --graph origin/master~2..command-parsing`.

The duplicate commits are easy to fix with interactive rebase (as opposed to normal). Run

	git checkout command-parsing
	git rebase --interactive origin/master

Then you'll get an editor buffer with this rebase-todo-list

	```
	pick acf96914 Fixed English mechanics: grammar, punctuation, and wording
	pick 0f5cb7ed Removed initial spacings
	pick 5e44376b Fixed English mechanics: grammar, punctuation, and wording
	pick e6d6a0e4 Removed initial spacings
	```

To remove the duplicate commits, simply remove the corresponding lines.
Additionally, to squash the second commit into the first, change the instruction to "fixup" (or "f")

	```
	pick acf96914 Fixed English mechanics: grammar, punctuation, and wording
	fixup 0f5cb7ed Removed initial spacings
	```

Reply:
Okay. I ran the rebase and then `git push --force`. I had to use --force, otherwise it was telling me to pull.